README.md

```diff
@@ -1,2 +1,73 @@
-# oop-cr-training
-This repo host a code for the tasks performed during the Code Reviewers training
+# OOP SCHOOL LIBRARY
+
+This is a school library Application.
+
+## Features
+
+- Added relationship between classroom and students
+- Added relationship person and rentals
+- Added relationship between class and books
+- Created book class with title and author variable instances
+- Created rental class with the date as a variable instance
+- Created classroom class and added label instance
+
+## Getting Started
+​
+To get a local copy up and running follow these simple example steps.
+​
+1. On the project GitHub page, navigate to the main page of the repository.
+2. Under the repository name, locate and click on a green button named `Code`. 
+3. Copy the project URL as displayed.
+4. If you're running the Windows Operating System, open your command prompt. On Linux, Open your terminal. 
+5. Change the current working directory to the location where you want the cloned directory to be made. Leave as it is if the current location is where you want the project to be. 
+6. Type git clone, and then paste the URL you copied in Step 3. <br>
+e.g. ``$ git clone https://github.com/od-c0d3r/oop-cr-training.git``
+7. Press Enter. Your local copy will be created. 
+8. Go into the folder you cloned in this step 7 on your terminal
+e.g.
+ ``$ cd Desktop/OOP-training``
+9. Check out in the feature branch
+
```

Comment:
- [ ] You did a great job with your README, but after a user checks out to the feature branch, you didn't provide further instructions on how they could run the app. It would be nice to specify the available commands and what they do.